src/native/libs/System.Native/pal_mount.c

```diff
@@ -91,6 +99,39 @@ int32_t SystemNative_GetAllMountPoints(MountPointFound onFound, void* context)
     return result;
 }
 
+#elif defined(__HAIKU__)
+    (void)MNTOPT_RO;
```

Comment:
Is this needed?

Reply:
Probably from an ancient build that failed due to some error related to this. I've removed the thing for now, let's see if the build fails again.